Adding an apartment-update-details view

tamirho opened the pull request:
- Adding apartment-update-details view.
- Adding two Form classes to handle the details update.
- Adding a template to update the details.

Fix #97 

**Note:**
The template needs to be redesigned in the future

tamirhouri committed:
Adding an apartment-update-details view

Adding apartment-update-details view.
Adding two Form classes to handle the details update.
Adding template to update the details.

Omeramsc reviewed (changes requested):
Please add tests for the view

tamirhouri committed:
Adding tests to aprtment-update view

- Adding tests to a login user in two scenarios: Owner and not Owner
- Alter the new_user fixture to work properly.

tamirho commented:
> Please add tests for the view

Please review again

Omeramsc commented on apartments/tests.py:
```diff
@@ -3,43 +3,46 @@
 import pytest
 
 
+@pytest.fixture
```
All of those fixtures exists also in `contacts/tests.py`, check if there is a way to use the same fixtures without creating duplicates.
I can see that it's not originated in this PR, so I'll approve it regardless, but create an issue and another PR to fix it (also check if it repeats in other places)

tamirho replied:
Ok, I will check it and fix it in another PR.

Micha546 reviewed (commented):
LGTMicha

DanielMy777 reviewed (commented):
Great!!
